vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc

```diff
@@ -46,6 +46,24 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
       - systemctl enable --now keepalived
       - chown {{ IMAGE_USERNAME }}:{{ IMAGE_USERNAME }} /home/{{ IMAGE_USERNAME }}/.kube/config
     files:
+      - path: /usr/local/bin/retrieve.configuration.files.sh
```

Comment:
We don't need this here. Its already baked in the image

Reply:
We find the issue in node image script in kubernetes version.

Reply:
Removed it and pushed a new version